cmd/cmdexecutor/cmdexecutor.go

```diff
@@ -50,7 +66,18 @@ func main() {
 	logrus.Infof("Running pod command executor: %v", version.Version)
 	flag.Parse()
 
-	if len(podList) == 0 {
+	podListSpecified := len(podList) > 0
+	selectorAndNamespaceSpecified := labelSelector != "" && namespace != ""
+
+	if labelSelector != "" && namespace == "" {
```

Comment:
nit: can use `selectorAndNamespaceSpecified` directly here

Reply:
Note the `namespace == ""` which is different from the `namespace != ""` used for `selectorAndNamespaceSpecified`.

I can introduce a variable `selectorWithoutNamespaceSpecified := labelSelector != "" && namespace == ""` if you think it might improve readability.